endpoints/openrtb2/auction.go

```diff
@@ -1816,7 +1816,9 @@ func (deps *endpointDeps) validateUser(req *openrtb_ext.RequestWrapper, aliases
 			return append(errL, errors.New(`request.user.ext.prebid requires a "buyeruids" property with at least one ID defined. If none exist, then request.user.ext.prebid should not be defined.`))
 		}
 		for bidderName := range prebid.BuyerUIDs {
-			if _, ok := deps.bidderMap[bidderName]; !ok {
+			normalizedCoreBidder, _ := openrtb_ext.NormalizeBidderName(bidderName)
+			coreBidder := normalizedCoreBidder.String()
+			if _, ok := deps.bidderMap[coreBidder]; !ok {
```

Comment:
Nitpick: in this PR here https://github.com/prebid/prebid-server/pull/3187/files#diff-785f5ab5ebde657757f3b3c34bf30a939782678d61bf1ea43f844c4f22a012d9R155:~:text=normalizeBidderName%20%20%20%20%20%20%20normalizeBidderName @SyntaxNode introduced a new function in endpoint deps. 
This validation function already has deps. Could you please pull latest changes from main branch and call normalize bidder name through the `normalizedCoreBidder, _ := deps.normalizeBidderName(bidderName)`. With this change code will be more consistent. 

Other than this looks good, local testing looks good too.

Reply:
Thank you, Veronika! Updated it in [49a8d0e](https://github.com/prebid/prebid-server/pull/3152/commits/49a8d0e5cbca2fb95c4f8af2b65f5868d0cd54f4)